docs/src/Checkbox.doc.js

```diff
@@ -93,6 +93,108 @@ function CheckboxExample() {
   />
 );
 
+card(
+  <Example
+    id="subtext"
+    description="Here is an example of a group of checkboxes with additional subtext applied."
+    name="Example: With Subtext"
+    defaultCode={`
+function CheckboxExample() {
+    const [checkedEn, setCheckedEn] = React.useState(false);
+    const [checkedSp, setCheckedSp] = React.useState(false);
+    const [checkedCh, setCheckedCh] = React.useState(false);
+
+  return (
+    <Box margin={-2}>
+      <Box padding={2}>
+        <Checkbox
+          checked={checkedEn}
+          id="english-info"
+          label="English"
+          subtext="USA, India, and Pakistan have the top number of English speakers "
+          name="languages"
+          onChange={({ checked }) => {
+            setCheckedEn(checked);
+          }}
+        />
+      </Box>
+      <Box padding={2}>
+        <Checkbox
+          checked={checkedSp}
+          id="spanish-info"
+          label="Spanish"
+          subtext="Mexico, Columbia, and Spain are the top three Spanish speaking countries"
+          name="languages"
+          onChange={({ checked }) => {
+            setCheckedSp(checked);
+          }}
+        />
+      </Box>
+      <Box padding={2}>
+        <Checkbox
+          checked={checkedCh}
+          id="chinese-info"
+          label="Chinese"
+          subtext="Chinese has two varieties: Cantonese and Mandarin"
+          name="languages"
+          onChange={({ checked }) => {
+            setCheckedCh(checked);
+          }}
+        />
+      </Box>
+    </Box>
+  );
+}
+`}
+  />
+);
+
+card(
+  <Example
+    id="images"
+    description="Here is an example of a group of checkboxes with images included. When including images, you can use the subtext property to clearly describe the information being presented by the image, or use the image's alt text to provide more context."
+    name="Example: With Images"
+    defaultCode={`
+function CheckboxExample() {
+    const [checkedEn, setCheckedEn] = React.useState(false);
+    const [checkedSp, setCheckedSp] = React.useState(false);
+    const [checkedCh, setCheckedCh] = React.useState(false);
+
+  return (
+    <Box margin={-2}>
+      <Box padding={1}>
+        <Checkbox
+          checked={checkedEn}
+          id="coral"
+          label="Coral"
+          subtext="Botanical art in coral and green"
+          image={<Box height={100} width={80} paddingX={1}><Image alt="" src="https://i.ibb.co/7bQQYkX/stock2.jpg" fit="container" naturalWidth={1} naturalHeight={1}/></Box>}
```

Comment:
@ayeshakmaz instead of setting the `paddingX` on the inner `Box`, is that something we should wrap around `image`?

```
{image && <Box paddingX={1}>{image}</Box>}
```

Related question: should there be a `maxWidth` on the image? @ashleyseto

Reply:
There's currently no size limit spec'd on the image size (according to the [figma](https://www.figma.com/file/EWNPd8z7knrIIrQrd8bAr6/Radio-%26-Checkbox-improvements?node-id=0%3A1)), so I figure if users need to use a Box anyway, then the padding can be up to the user when they set that width and height (though I do recognize that that can produce some inconsistencies) We can at least add guidance in the docs around what padding might be most optimal?